app/models/user.rb

```diff
@@ -18,4 +18,8 @@ def self.from_omniauth(auth)
       user.skip_confirmation!
     end
   end
+
+  def self.default_genders
```

Comment:
In theory, this should be placed in some kind of presenter or decorator instead of the model. The `User` model already presents the `genders` in some specific way, displaying the same data in a different way is not a model responsibility.